source/blog/articles/Tutorials/2018-10-01-workbench_on_gcp.html.markdown

```diff
@@ -8,183 +8,152 @@ tags: [ Rubykube, Tutorials ]
 
 ---
 
-<p>
-<strong>Requirements:</strong>
-</p>
 
-<ul class="ml-2">
-<li>* Experience with Google Cloud Platform.</li>
-<li>* Google Cloud Project.</li>
-</ul>
-
-<h2 class="mt-4 mb-3">Create Virtual Machine</h2>
-
-<ul class="ml-2">
-    <p>* Open Google Cloud Console.</p>
-    <p>* On the left sidebar find <strong>Compute Engine -> VM instances.</strong></p>
-    <p>* At the top of the page click <strong>CREATE INSTANCE.</strong></p>
-    <p>* Change name of the instance from <em>instance-1</em> to a better name.</p>
-    <p>* Choose <strong>region</strong> and <strong>zone</strong>, that fits your needs best.</p>
-    <p>* Select machine type, recommended is <strong>n1-standart-8</strong>, required is <strong>n1-standart-4</strong>.</p>
-    <p>* Increase size of the boot disk, recommended size is <strong>60GB</strong>, required is <strong>45GB</strong>.</p>
-    <p>* Don't change default OS image: <em>Debian GNU/Linux 9 (stretch)</em>.</p>
-    <p>* Check two checkboxes under the <strong>Firewall</strong> section.
-        <ul class="ml-4">
-          <li>
-            <input type="checkbox" id disabled class="task-list-item-checkbox" checked>
-             Allow HTTP traffic
-          </li>
-          <li>
-            <input type="checkbox" id disabled class="task-list-item-checkbox" checked>
-             Allow HTTPS traffic
-          </li>
-        </ul>
-    </p>
-    <p>* <strong>Double check</strong> your settings before creating.</p>
-    <p>
-      <img class="col-md-12" src="https://camo.githubusercontent.com/087e942149862f48dc0177d051842371e62a2254/68747470733a2f2f64726976652e676f6f676c652e636f6d2f75633f69643d314c536c746d4944744c714b786d6b435a7030324f485a654d375f637630595456" alt="Settings before VM Creation">
-    </p>
-    <p>* Click <strong>Create</strong> button at the bottom of the page.</p>
-</ul>
-
-<h2 class="mt-4 mb-3">Post creation steps</h2>
-<p>Here is your created vm. In the green rectangle you will find public IP of your virtual machine.</p>
+**Requirements:**
+
+---
+- <span>*</span> Experience with Google Cloud Platform.
+- <span>*</span> Google Cloud Project.  
+         
+<span></span>        
```

Comment:
Do we need this empty span?
There are a few more later in doc

Reply:
They are for increasing the padding between blocks of text

Reply:
@DmytroYesin lets just put <br/> tags